WPMedia/fusion-news-theme-blocks
# THEMES-1916: Update Author images to use signed resizer images if available.

## Description

#### Jira Ticket: [THEMES-1916](https://arcpublishing.atlassian.net/browse/THEMES-1916)

Adding resizer support to author blocks and updating tests

## Test Steps
1. Checkout this branch `git checkout themes-1916`
2. Run fusion repo with linked blocks `npx fusion start`
3. Add the author blocks to your page and examine the image component to see that it's using srcset


[THEMES-1916]: https://arcpublishing.atlassian.net/browse/THEMES-1916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
